packages/dnlib.vm/dnlib.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>dnlib.vm</id>
-    <version>4.0.0.20250219</version>
+    <version>4.0.0.20250505</version>
     <authors>0xd4d</authors>
-    <description>.NET module/assembly reader/writer library</description>
+    <description>Dnlib is a .NET module/assembly reader/writer library.</description>
```

Use the official tool name from the official home page (here and for other tools):
```suggestion
    <description>dnlib is a .NET module/assembly reader/writer library.</description>
```